src/rules.js

```diff
@@ -1903,6 +1903,80 @@ module.exports = [
     },
     /**
      ==================================================================
+     TRANSITION
+     ==================================================================
+     */
+    {
+        type: 'pattern',
+        id: 'transition',
+        name: 'Transition',
+        prefix: '.Trs-',
+        properties: ['transition'],
+        allowCustom: true,
+        allowCustomAutoSuffix: false,
+        allowSuffixToValue: true
```

Comment:
I don't know, is that really the way we want to go with these suffixes?  Seems like it'd get unnecessarily complex very quickly - some of these take a lot of values...